### PR TITLE
Nav bar in layout

### PR DESCRIPTION
**Pull request by perryr16**
Moved the css around so the nav bar ONLY exists in views/layouts/application.html.erb

### CODE REVIEW

File: app/views/pets/edit.html.erb [hunk 1]
```diff
@@ -1,14 +1,4 @@
...
-  <img src="https://wallpaperaccess.com/full/783057.jpg" alt="kitten">
-</div>
-
 <div class="content">
-  <div class="main_menu">
-    <a href="/">Main</a>
-    <a href="/shelters">Shelters</a>
-    <a href="/shelters/new">New Shelter</a>
-    <a href="/pets">Pets</a>
-  </div>
```
Comment (perryr16) [inline app/views/pets/edit.html.erb:2]: this is the same for all views. now the navbar and background image are in the views/layouts/application

File: spec/features/favorites/index_spec.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe "Favorite Indicator", type: :feature do
+
```
Comment (perryr16) [inline spec/features/favorites/index_spec.rb:4]: all hail TDD

File: app/views/pets/edit.html.erb (hunk 1 above)
Comment (muydanny) [reply app/views/pets/edit.html.erb:2]: beautiful

**Pull request merged**